generator/src/main/java/com/scottlogic/deg/generator/generation/ReductiveDataGeneratorMonitor.java

```diff
@@ -14,4 +16,6 @@ void rowSpecEmitted(
 
     void fieldFixedToValue(Field field, Object current);
     void unableToStepFurther(ReductiveState reductiveState);
+    void noValuesForField(ReductiveState reductiveState);
```

Comment:
Introduced to aid debugging, but do not NEED to be kept. More effort to remove at this stage so suggest leaving.